packages/extension-task-item/src/task-item.ts

```diff
@@ -65,7 +65,11 @@ export const TaskItem = Node.create<TaskItemOptions>({
       checked: {
         default: false,
         keepOnSplit: false,
-        parseHTML: element => element.getAttribute('data-checked') === 'true',
+        parseHTML: element => {
+          const dataChecked = element.getAttribute('data-checked')
+
+          return dataChecked != null && dataChecked !== 'false'
+        },
```

Comment:
I think I would do `element.getAttribute('data-checked') === '' || element.getAttribute('data-checked') === 'true'` To be explicit about what we accept

Reply:
I don't have a opinion for this. updated: https://github.com/ueberdosis/tiptap/pull/5366/commits/0f4c85d5eda54481a576c97c4df26335a56d4f70